api/BaseAudioContext.json

```diff
@@ -113,15 +99,20 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/BaseAudioContext/createAnalyser",
           "support": {
-            "chrome": {
-              "version_added": "10",
-              "prefix": "webkit"
-            },
+            "chrome": [
+              {
+                "version_added": "56"
```

Comment:
This runs into https://github.com/mdn/browser-compat-data/issues/3463. `createAnalyser` existed unprefixed much earlier, but on the `AudioContext` interface.

I think what we'll have to do is an entry with a note for the versions of Chrome before `BaseAudioContext` was added that instead had `createAnalyser` on `AudioContext`.